lib/ovirt_metrics.rb

```diff
@@ -59,7 +59,8 @@ def self.host_realtime(host_id, start_time = nil, end_time = nil)
     host_realtime_metrics_to_hashes(metrics, nic_metrics)
   end
 
-  private
+  # The methods below this line are PRIVATE not meant to be used out of the scope
```

Comment:
you can make class methods private with `private_class_method`
but as you said in your comment, I'd do this in a follow up PR and leave the statement here